rqpy/plotting/_iv_didv_tools_plotting.py

```diff
@@ -165,8 +165,8 @@ def _plot_rload_rn_qetbias(IVanalysisOBJ, lgcsave, xlims_rl, ylims_rl,
     )
     axes[1].grid(True, linestyle='dashed')
     axes[1].set_title('Rtotal vs Vbias', fontsize=14)
-    axes[1].set_ylabel(r'$R_{N} + R_ℓ$ [mΩ]', fontsize=14)
-    axes[1].set_xlabel(r'$V_{bias}$ [μV]', fontsize=14)
+    axes[1].set_ylabel(r'$R_N + R_\ell \, [\mathrm{m}\Omega]$', fontsize=14)
+    axes[1].set_xlabel(r'$V_\text{bias} \, [\mu\mathrm{V}]$', fontsize=14)
```

Comment:
The `\text` function I don't think is supported in matplotlib LaTeX
```suggestion
    axes[1].set_xlabel(r'$V_\mathrm{bias} \, [\mu\mathrm{V}]$', fontsize=14)
```